iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -1547,69 +1652,6 @@ public void disableDeviceAfterOpenAndAfterRegistration() throws Exception
         }
     }
 
-    // If you register a disabled device to a multiplexed connection that hasn't opened yet, the open call should succeed
-    // but the disabled device's session should drop shortly afterwards and the other devices on the multiplexed connection
-    // should be unaffected.
-    @ContinuousIntegrationTest
-    @Test
-    public void disableDeviceBeforeOpen() throws Exception
```

Comment:
This test depended on inconsistent service behavior, so I'm removing it. Basically, it was random if opening a mux client with a disabled device would cause the client to fail on open, or fail on connection status callback for the disabled device. Since we can't predict how fast the service will respond on these device disabled cases, it doesn't make sense to have a test around it